pkg/bootkube/bootstrap.go

```diff
@@ -0,0 +1,84 @@
+package bootkube
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/kubernetes-incubator/bootkube/pkg/asset"
+)
+
+// CreateBootstrapControlPlane seeds static manifests to the kubelet to launch the bootstrap control
+// plane.
+func CreateBootstrapControlPlane(assetDir string, podManifestPath string) error {
+	UserOutput("Running temporary bootstrap control plane...\n")
+
+	// Make secrets temporarily available to bootstrap cluster.
+	if err := os.RemoveAll(asset.BootstrapSecretsDir); err != nil {
+		return err
+	}
+	if err := os.Mkdir(asset.BootstrapSecretsDir, os.FileMode(0755)); err != nil {
+		return err
+	}
+	secretsDir := filepath.Join(assetDir, asset.AssetPathSecrets)
+	secrets, err := ioutil.ReadDir(secretsDir)
+	if err != nil {
+		return err
+	}
+	for _, secret := range secrets {
+		if err := copyFile(filepath.Join(secretsDir, secret.Name()), filepath.Join(asset.BootstrapSecretsDir, secret.Name()), true); err != nil {
```

Comment:
Is it safe to assume that bootkube and the kubelet are always both running as root (seems to be the case in container linux)? If so I think we can lock this down to 0700.